jamf_protect_api/scripts/python/update_alert_status.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python3
+
+# This example Python script updates the status of alert(s) (New, InProgress, or Resolved).
+
+# The script does the following:
+#
+# - Obtains an access token.
+# - Updates the status of the provided alert(s), using the
+#   updateAlerts mutation.
+
+# Keep the following in mind when using this script:
+#
+# - You must define the PROTECT_INSTANCE, CLIENT_ID, and PASSWORD variables to
+#   match your Jamf Protect environment. The PROTECT_INSTANCE variable is your
+#   tenant name (eg. your-tenant), which is included in your tenant URL (eg.
+#   https://your-tenant.protect.jamfcloud.com).
+# - There are two required arguments that must be passed when running the script,
+#   -u/--uuid and -s/--status.
+# - Maximum number of uuids that can be passed is 100.
+# - This script requires the 3rd party Python library 'requests'
+
+import requests
+import argparse
+
+PROTECT_INSTANCE = ""
+CLIENT_ID = ""
+PASSWORD = ""
+
+
+def get_access_token(protect_instance, client_id, password):
+    """Gets a reusable access token to authenticate requests to the Jamf
+    Protect API"""
+
+    token_url = f"https://{protect_instance}.protect.jamfcloud.com/token"
+
+    payload = {
+        "client_id": client_id,
+        "password": password,
+    }
+
+    resp = requests.post(token_url, json=payload)
+    resp.raise_for_status()
+
+    resp_data = resp.json()
+    print(
+        f"Access token granted, valid for {int(resp_data['expires_in'] // 60)} minutes."
+    )
+
+    return resp_data["access_token"]
+
+
+def make_api_call(protect_instance, access_token, query, variables=None):
+    """Sends a GraphQL query to the Jamf Protect API, and returns the
+    response."""
+    if variables is None:
+        variables = {}
+    api_url = f"https://{protect_instance}.protect.jamfcloud.com/graphql"
+    payload = {"query": query, "variables": variables}
+    headers = {"Authorization": access_token}
+    resp = requests.post(
+        api_url,
+        json=payload,
+        headers=headers,
+    )
+    resp.raise_for_status()
+    return resp.json()
+
+
+UPDATE_ALERT_STATUS_QUERY = """
+    mutation updateAlerts($input: AlertUpdateInput!){
+        updateAlerts(input: $input) {
+            items {
+                status
+            }
+        }
+    }
+"""
+
+
+def create_args():
+    parser = argparse.ArgumentParser(
+        description="Given an Alert(s) UUID(s), set the current status."
+    )
+    parser.add_argument(
+        "-u",
+        "--uuid",
```

Comment:
Again, just my singular/plural nitpickiness
```suggestion
        "--uuids",
```

Reply:
done